src/commands/app/create.ts

```diff
@@ -96,7 +97,15 @@ export default class Create extends BaseCommand<typeof Create> {
           message: this.messages.CONFIRM_CLONE_BOILERPLATE,
         }))
       ) {
-        await this.boilerplateFlow();
+        const boilerplate = await selectedBoilerplate();
+        if (boilerplate) {
+          this.sharedConfig.boilerplateName = boilerplate.name
+            .toLowerCase()
+            .replace(/ /g, "-");
```

Comment:
@harshithad0703 , Please review the previous flow. We were asking user to input app name using the prompt; you don't have to create it. Here you can check the condition **contentstack-apps-cli/src/util/inquirer.ts**